lib/globalize.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
@0candy I noticed there is no licensing text at the top of this file. I assume you'll have to take care of this as part of your globalization work?